README.md

```diff
@@ -79,6 +84,21 @@ const YourComponent = () => {
 };
 ```
 
+Optional configuration is available with `config` prop. It will take priority over environment variables.
+
+```jsx
+<FlagProvider
+  config={{
+    url: "http://localhost:4242/api/frontend", // this will override NEXT_PUBLIC_UNLEASH_FRONTEND_API_URL
+    clientKey: "<Frontend_API_token>", // NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN
+    appName: "nextjs", // NEXT_PUBLIC_UNLEASH_APP_NAME
```

Comment:
How about adding an "overrides" to each line, so that it clear what it does?

```suggestion
    url: "http://localhost:4242/api/frontend", // replaces NEXT_PUBLIC_UNLEASH_FRONTEND_API_URL
    clientKey: "<Frontend_API_token>", // replaces NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN
    appName: "nextjs", // replaces NEXT_PUBLIC_UNLEASH_APP_NAME
```